fau-klue/pandas-association-measures
# upgrade to 0.1.4

- AMs
  - bugfix in MI
  - signed version of log-likelihood
  - add log-ratio
- test data
  - add UCS gold
- naming conventions
  - follow Evert (2004: 36) *or* contingency table
- usability:
  - calculate observed and expected frequencies on the fly for AMs
  - add inplace operations
- extend README

Fixes https://github.com/fau-klue/pandas-association-measures/issues/12
